apps/mull-api/src/app/post/inputs/post-reaction.input.ts

```diff
@@ -0,0 +1,8 @@
+import { Field, InputType, Int } from '@nestjs/graphql';
+import { PostReaction } from '../../entities';
+
+@InputType()
+export class PostReactionInput implements Partial<PostReaction> {
+  @Field(/* istanbul ignore next */ () => Int)
+  id: number;
+}
```

Comment:
💡 I don't think this needs to be in a separate file, it could be moved to `post.input.ts`

Reply:
Since it has a separate entity file I thought I would keep it consistant